src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -168,7 +168,7 @@ protected override void ProcessRecord()
 
                             if (validationResults.Message != null)
                             {
-                                ErrorRecord errorRecord = new(exception, "InvalidJsonAgainstSchemaDetailed", ErrorCategory.InvalidData, null);
+                                ErrorRecord errorRecord = new(exception, "InvalidJsonAgainstSchemaDetails", ErrorCategory.InvalidData, null);
```

Comment:
I wonder why it is not InvalidJsonAgainstSchemaDetailed too

Reply:
It is. I think you're looking at the previous commit.